web-client/src/presenter/actions/setPetitionFormValueAction.ts

```diff
@@ -0,0 +1,17 @@
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const setPetitionFormValueAction = ({
```

Comment:
This action is really similar to `setFormValueAction.ts` above. Maybe they could be consolidated with some adjustments.